docs/03-Plugins/refdiff.md

```diff
@@ -84,8 +84,12 @@ make
 make install
 ```
 
-### MacOs
-
+### MacOS
+1. HomeBrew install
```

Comment:
HomeBrew will install libgit2 1.4.3, I think macport is better?

Reply:
Yes, macports is better, it allows better version control